rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/FileMetadata.java

```diff
@@ -99,6 +101,22 @@ public FileMetadata(String remotePath,
      */
   }
 
+  private void createLocalFiles()
```

Comment:
add comment to method signature that BKS is responsible for creating new files (data + mdfiles) via FileMetadata in all cases. RRCs should never create new files, if they ever hit a case of FNF then they should fail the request as it points to invalidations.

Reply:
I see you have it already. Reword it so that the message is strong that this responsibility shouldnt change and move the comment before method name